main.py

```
@@ -0,0 +1,126 @@
+    
+    # JSON 추출
+    json_match = re.search(r'\{[^}]*\}', result)
+    if json_match:
+        try:
+            json_str = json_match.group()
+            parsed = json.loads(json_str)
+            clean_result = {
+                "isCurse": parsed.get("isCurse", False),
+                "words": parsed.get("words", [])
+            }
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)